watertap/core/util/initialization.py

```diff
@@ -136,3 +138,44 @@ def assert_no_degrees_of_freedom(blk):
 
     """
     check_dof(blk, True)
+
+
+def interval_initializer(blk, feasibility_tol=1e-6):
+    """
+    Improve the initialization of ``blk`` utilizing interval arithmetic.
+
+    Keyword Arguments:
+        blk : block to initialize
+        feasibility_tol : tolerance to use for FBBT (default: 1e-6)
+
+    Returns:
+        None
+
+    """
+
+    bound_cache = ComponentMap()
+
+    for v in blk.component_data_objects(Var, active=True, descend_into=True):
+        bound_cache[v] = v.bounds
+
+    fbbt(blk, feasibility_tol=feasibility_tol, deactivate_satisfied_constraints=False)
+
+    for v, bounds in bound_cache.items():
+        if v.value is None:
+            # set to the bound nearer 0
+            v.set_value(0, skip_validation=True)
```

Comment:
For clarity, why is the `skip_validation` required here? Is it because the bounds value is a float but `v` might have units?

Reply:
Mostly performance, but the validation could also fail in other ways I do not care about, like what you mention.

The bounds will always be a float.